style/README.md

```diff
@@ -40,7 +40,9 @@ Formatting
 * Don't misspell.
 * Don't vertically align tokens on consecutive lines.
 * If you break up an argument list, keep the arguments on their own lines and
-  closing parenthesis on its own line.
+  closing parenthesis on its own line. [Example 1][multi-line arguments]
+  [Example 2][long method signature]
+  
```

Comment:
I don't think you need this blank line, it causes a paragraph tag to be added to this li.